src/router/appLoadingManagement.routerPlugin.js

```diff
@@ -52,12 +47,12 @@ const appLoadingManagementRouterPlugin = (router, store) => {
                     isLegacyUrl,
                     ...dispatcher,
                 })
+                unRegisterRouterHook()
```

Comment:
What made it not working well keeping it under in the store subsribe?

Reply:
Nothing i just thought it would be wise to stop this plug-in sooner rather than later